scripts/install.py

```diff
@@ -228,14 +230,15 @@ def q_final_confirm(run_name, yes_statcords, yes_model_params):
     return show_yes_no_question()
 
 
-def action(sim_dir, event_name, run_name, run_dir, vel_mod_dir, srf_dir, srf_stoch_pairs, params_vel_path,
-           stat_file_path, vs30_file_path, vs30ref_file_path, MODEL_LAT, MODEL_LON, MODEL_ROT, hh, nx, ny, nz, sufx,
-           sim_duration, flo, vel_mod_params_dir, yes_statcords, yes_model_params,dt = default_dt, hf_dt = default_hf_dt):
-    lf_sim_root_dir, hf_dir, bb_dir, figures_dir = os.path.join(sim_dir, "LF"), os.path.join(sim_dir,
-                                                                                             "HF"), os.path.join(
-        sim_dir, "BB"), os.path.join(sim_dir, "Figures")
+def create_params_dict(version, sim_dir, event_name, run_name, run_dir, vel_mod_dir, srf_dir, srf_file, stoch_file,
+                       params_vel_path, stat_file_path, vs30_file_path, vs30ref_file_path, MODEL_LAT, MODEL_LON,
+                       MODEL_ROT, hh, nx, ny, nz, sufx, sim_duration, flo, vel_mod_params_dir, yes_statcords,
+                       yes_model_params, dt=default_dt, hf_dt=default_hf_dt):
```

Comment:
Parameters flo, hf_dt and srf_file don't appear to be used in the function?

Reply:
good spot. flo and hf_dt now in dummpy template; srf_file is used.